packages/blade/src/components/ChatMessage/_decision.md

```diff
@@ -0,0 +1,85 @@
+# ChatMessage Decisions
+
```

Comment:
You can add annotations image from figma cover of this component like we do in other API decisions